HelloWorld/Main.java

```diff
@@ -4,6 +4,6 @@
 {
     public static void main(String[] args)
     {
-       System.out.println("Hello world");
+       System.out.println("Hello world!!!");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.alumni-services-001.com/coding_rules#rule_key=squid%3AS106)